api/clever/__init__.py

```diff
@@ -44,11 +49,17 @@
     TITLE_I_NCES_IDS = json.loads(json_data)
 
 
-class CleverAuthenticationAPI(OAuthAuthenticator):
+class CleverAuthenticationAPI(OAuthAuthenticationProvider):
 
-    NAME = 'Clever'
     URI = "http://librarysimplified.org/terms/auth/clever"
-    METHOD = "http://librarysimplified.org/authtype/Clever"
+
+    # TODO: I think this can be removed because
+    # "http://librarysimplified.org/authtype/OAuth-with-intermediary"
+    # should be good enough.
+    # METHOD = "http://librarysimplified.org/authtype/Clever"
```

Comment:
I've taken this out but we might need to add it back in if our deployed Open Ebooks client is looking for it.